storage/file_persister.go

```diff
@@ -25,11 +25,11 @@ func (l *LocalFilePersister) Persist(_ context.Context, path string, data io.Rea
 	cp := filepath.Clean(path)
 
 	dir := filepath.Dir(cp)
-	if err = os.MkdirAll(dir, 0o755); err != nil {
+	if err = os.MkdirAll(dir, 0o755); err != nil { //nolint:forbidigo,gosec
 		return fmt.Errorf("creating a local directory %q: %w", dir, err)
 	}
 
-	f, err := os.OpenFile(cp, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0o600)
+	f, err := os.OpenFile(cp, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0o600) //nolint:forbidigo
```

Comment:
Not related to the PR, but this probably should be refactored, it shouldn't use directly os, right?

Reply:
Yes - I have a list of things that shoudl be fixed, but rewritign browser as part of getting linter issues fixed turned out to be way too much work.

Reply:
This should only occur when running tests locally, when running in the cloud the env is set so that the remote_persister is used.. When we wrote this there was no way to write arbitrary files using the k6 implemented FS API (i think).